addons/stock/report/report_stock_forecasted.py

```diff
@@ -114,6 +115,10 @@ def _prepare_report_line(self, quantity, move_out=None, move_in=None, replenishm
         move_in_id = move_in.id if move_in else None
         move_out_id = move_out.id if move_out else None
 
+        timezone = pytz.timezone(self.env.context['tz']) if 'tz' in self.env.context else pytz.utc
+        move_in_date = move_in.date.astimezone(timezone) if move_in else None
```

Comment:
maybe we could combine the two `... if ... else ...` and have False here

```suggestion
        move_in_date = format_date(self.env, move_in.date.astimezone(timezone)) if move_in else False
```
and you can just use move_in_date after.

As an side note, I think it's possible to use fields.Datetime.context_timestamp to remove part of the code, this should (not tested) be equivalent to current line :

```suggestion
        move_in_date = fields.Datetime.context_timestamp(self, move_in) if move_in else False
```